### PR TITLE
.DEREK.yml: add myself

### PR DESCRIPTION
**Pull request by kolyshkin**
... so finally I'll be able to talk to a bot rather than a real person in order to restart a CI job ;)

### CODE REVIEW

File: .DEREK.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ curators:
   - ehazlett
   - fntlnz
   - gianarb
+  - kolyshkin
```
Comment (thaJeztah) [inline .DEREK.yml]: Might want to add yourself to the "curators" list as well (like https://github.com/moby/moby/commit/46c713ab5ad20b97de01537a8092d2e747f9ad0e)

Comment (kolyshkin) [reply .DEREK.yml]: yep, done

**Review by yongtang — APPROVED** [review]
LGTM 👍

**Review by vdemeester — APPROVED** [review]
LGTM :tiger:

**Review by thaJeztah — APPROVED** [review]
LGTM 👍 🤗

**Pull request merged**